bin/hdfcoinc/pycbc_page_injtable

```diff
@@ -26,15 +28,41 @@ else:
     title = "Found Injections"
     found = f['found_after_vetoes']
     idx = found['injection_index'][:]
-    tdiff = (found['time1'][:] - found['time2'][:]) * 1000
-    tdiff_str = '%s - %s time (ms)' % (f.attrs['detector_1'], f.attrs['detector_2'])
+    if('detector_1' in f.attrs):
+        tdiff = (found['time1'][:] - found['time2'][:]) * 1000
+        tdiff_str = '%s - %s time (ms)' % (f.attrs['detector_1'], f.attrs['detector_2'])
+        ids = {f.attrs['detector_1']: found['trigger_id1'][:],
+               f.attrs['detector_2']: found['trigger_id2'][:],}
+        found_cols = [tdiff, found['stat'], found['ifar'], found['ifar_exc']]
+        found_names = [tdiff_str, 'Ranking Stat.', 'Inc. IFAR (yrs)', 'Exc. IFAR']
+        found_formats =  ['##.##', '##.##', '##', '##']
+
+    else:
+        detectors = ['L1','H1','V1','K1','I1']
+        keys = f['found_after_vetoes'].keys()
+        detectors_used = []
+        found = f['found_after_vetoes']
+        for det in detectors:
+            if(det in keys):
+                detectors_used.append(det)
+        detectors_two_combinations= numpy.array(list(combinations(detectors_used,2)))
+        tdiff = []
+        tdiff_str = []
+        tdiff_format =[]
+        for i in range(len(detectors_two_combinations)):
```

Comment:
as i is just a number used to reference detectors_two_combinations in this loop, you should be able to loop over detectors_two_combinations directly (I think)